mailbox/api/src/main/java/org/apache/james/mailbox/model/MessageAttachment.java

```diff
@@ -76,7 +76,7 @@ public MessageAttachment build() {
                 isInline = false;
             }
             if (isInline && !cid.isPresent()) {
-                throw new IllegalStateException("'cid' is mandatory for inline attachments");
+                isInline = false;
```

Comment:
there's something wrong : I can set isInline to true and have it set to false after build.
That probably means we are not using the right abstraction